test/performance/common/requirements.txt

```diff
@@ -0,0 +1 @@
+kubernetes==31.0.0
```

Comment:
This seems a bit aggressive to me. See https://kubernetes.io/releases/version-skew-policy/#kubectl and #kubelet , and compare this repo's `go.mod`. Clients are generally allowed to be a few versions older than the apiserver but not newer. I see https://github.com/kubernetes-client/python?tab=readme-ov-file#compatibility, in which the "+-" is a less precise statement of these relationships.

Reply:
ok, understood. I revert it back to old version:

``
kubernetes==30.1.0
``